### ISSUE TITLE
print_stream_info(id) returns TypeError: can't pickle _thread.lock objects

### ISSUE THREAD

**Issue by brbatv** [oliver-zehentleitner/unicorn-binance-websocket-api#39]
Hello
I'm getting a bug when using the manager inside a class. 
I'm on linux mint 18.3, using python 3.7.1 with a venv, using unicorn 1.10.1

Please see the code : 
```python
from unicorn_binance_websocket_api.unicorn_binance_websocket_api_manager import BinanceWebSocketApiManager

class LivePortfolio(object):
    """
    Live Binance Portfolio
    """

    def __init__(self):
        self.manager = BinanceWebSocketApiManager()
        self.stream_ids = {'asset_price': None}
        self.asset_list = ['BTC', 'ETH']
        self.available_market_list = ['ETHBTC', 'ETHUSDT', 'BTCUSDT']

        self.__init_websocket_subs(self.asset_list)

    def __init_websocket_subs(self, asset_list):
        if self.manager.get_active_stream_list() is False:
            markets = []
            available_market_list = self.available_market_list
            for asset in asset_list:
                if asset + 'USDT' in available_market_list:
                    markets.append((asset + 'USDT').lower())
                if asset + 'BTC' in available_market_list:
                    markets.append((asset + 'BTC').lower())

            self.stream_ids['asset_price'] = self.manager.create_stream(channels=['kline_1m'], markets=markets)


if __name__ == '__main__':
    l = LivePortfolio()
    l.manager.print_stream_info(l.stream_ids['asset_price'])
```

launching this script is giving the following error : 
Traceback (most recent call last):

> Traceback (most recent call last):
  File "unicorn_bug.py", line 32, in <module>
    l.manager.print_stream_info(l.stream_ids['asset_price'])
  File "/home/brbatv/PycharmProjects/thebot/venv/lib/python3.7/site-packages/unicorn_binance_websocket_api/unicorn_binance_websocket_api_manager.py", line 1648, in print_stream_info
    stream_info = self.get_stream_info(stream_id)
  File "/home/brbatv/PycharmProjects/thebot/venv/lib/python3.7/site-packages/unicorn_binance_websocket_api/unicorn_binance_websocket_api_manager.py", line 1298, in get_stream_info
    temp_stream_list = copy.deepcopy(self.stream_list)
  File "/usr/lib/python3.7/copy.py", line 150, in deepcopy
    y = copier(x, memo)
  File "/usr/lib/python3.7/copy.py", line 240, in _deepcopy_dict
    y[deepcopy(key, memo)] = deepcopy(value, memo)
  File "/usr/lib/python3.7/copy.py", line 150, in deepcopy
    y = copier(x, memo)
  File "/usr/lib/python3.7/copy.py", line 240, in _deepcopy_dict
    y[deepcopy(key, memo)] = deepcopy(value, memo)
  File "/usr/lib/python3.7/copy.py", line 169, in deepcopy
    rv = reductor(4)
TypeError: can't pickle _thread.lock objects

I don't know if the problem is coming from my usage of the manager or the manager itself. 
Also the connexion seems to be active (
Thank you very much for your help :)

**Comment by brbatv** [comment]
Ok, I just realized version 1.10.2 was out 4 hours ago and upgrading did fix this issue :)

**Comment by oliver-zehentleitner** [comment]
> Ok, I just realized version 1.10.2 was out 4 hours ago and upgrading unicorn did fix this issue even though I don't really know why :)

I was reading it in the subway and i assumed you just need an upgrade :)

In 1.10.1 i was trying to fix issue https://github.com/oliver-zehentleitner/unicorn-binance-websocket-api/issues/37 in method `get_stream_info()` which is used by `print_stream_info()` but it wasn’t a good solution and triggered a new bug - i fixed it with 1.10.2.

best regards!
